webpack.config.js

```diff
@@ -160,6 +161,10 @@ module.exports = (env = process.env.NODE_ENV || 'development') => {
           use: ['val-loader'],
           enforce: 'pre',
         },
+        {
+          test: /\.gen\.js$/u,
+          use: [{loader: 'babel-loader', options: babelLoaderConfig}],
+        },
```

Comment:
Can you explain why we need this, given the next rule should also match? Is it pre loader vs. normal loader?